hrjobcontract/js/test/services/contract-revision-list_test.js

```diff
@@ -0,0 +1,92 @@
+          "health_revision_id": "159",
+          "hour_revision_id": "159",
+          "leave_revision_id": "159",
+          "pay_revision_id": "159",
+          "pension_revision_id": "159",
+          "role_revision_id": "159",
+          "deleted": "0",
+          "editor_name": "admin@example.com"
+        }
+      ]
+    };
+
+    // Mocking http calls
+    $httpBackend.when('GET','/index.php?q=civicrm/ajax/rest&action=get&entity=HRJobContractRevision&json=%7B%22deleted%22:0,%22options%22:%7B%22limit%22:0%7D,%22sequential%22:1,%22jobcontract_id%22:%22159%22%7D').respond(responseData);
+    $httpBackend.when('GET','/index.php?q=civicrm/ajax/rest&action=get&entity=HRJobContract&json=%7B%22sequential%22:1,%22deleted%22:0%7D').respond({});
+    $httpBackend.when('GET','/index.php?q=civicrm/hrjobcontract/file/list&entityID=159&entityTable=civicrm_hrjobcontract_details').respond({});
+    $httpBackend.when('GET','/index.php?q=civicrm/ajax/rest&action=getsingle&entity=HRJobContractRevision&json=%7B%22sequential%22:1,%22id%22:%22159%22,%22return%22:%5B%22details_revision_id.position%22,%22details_revision_id.location%22,%22hour_revision_id.hours_type%22,%22pay_revision_id.pay_scale%22,%22pay_revision_id.pay_annualized_est%22,%22pay_revision_id.pay_currency%22%5D%7D').respond({});
+    $httpBackend.whenGET(/views.*/).respond({});
+
+    $httpBackend.expectGET('/index.php?q=civicrm/ajax/rest&action=get&entity=HRJobContractRevision&json=%7B%22deleted%22:0,%22options%22:%7B%22limit%22:0%7D,%22sequential%22:1,%22jobcontract_id%22:%22159%22%7D');
+  }));
+
+  describe('ContractRevisionList', function () {
+
+    describe('check if', function () {
+      beforeEach(function () {
+        spyOn(ContractRevisionList,'fetchRevisions');
+      })
+
+      it('defines fetchRevisions()', function () {
+        expect(ContractRevisionList.fetchRevisions).toBeDefined();
+      });
+
+      it('calls fetchRevisions functiion with argument', function () {
+        ContractRevisionList.fetchRevisions('159');
+        expect(ContractRevisionList.fetchRevisions).toHaveBeenCalled();
+        expect(ContractRevisionList.fetchRevisions).toHaveBeenCalledWith('159');
+      });
+    });
+
+    describe('when calling fetchRevisions()', function () {
+      it('return the revisionList and revisionDataList for a contract', function (done) {
+        ContractRevisionList.fetchRevisions('159').then(function (result) {
+          // Expecting data in revisionList
+          expect(result.revisionList.length).toEqual(1);
+          expect(result.revisionList[0].created_date).toBe('2017-02-14 04:30:29');
+          expect(result.revisionList[0].effective_date).toBe('2017-02-13');
+          expect(result.revisionList[0].modified_date).toBe('2017-02-14 04:30:31');
+
+          // Expecting data in revisionDataList
+          expect(result.revisionDataList.length).toBe(1);
+          expect(result.revisionDataList[0].revisionEntityIdObj.created_date).toBe('2017-02-14 04:30:29');
```

Comment:
Do not hardcode the value, 
`toBe(responseData. created_date)`
Check all instances